streamline_training.py

```diff
@@ -211,26 +276,23 @@ def __export_to_bucket():
     """ Streamline the preprocessing, training, and exporting process of the LaserTagger model using the hyperparameters
     in the command line arguments.
     
-    usage: streamline_training.py [-h] [-vocab_size VOCAB_SIZE]
-                              [-train_batch_size TRAIN_BATCH_SIZE]
-                              [-learning_rate LEARNING_RATE]
-                              [-num_train_epochs NUM_TRAIN_EPOCHS]
-                              [-warmup_proportion WARMUP_PROPORTION]
-                              [-max_input_examples MAX_INPUT_EXAMPLES]
-                              [-train] [-export] [-use_tpu] [-gbucket GBUCKET]
-                              model_output_dir abs_path_to_lasertagger
-                              abs_path_to_bert training_file tuning_file
-    
+    usage: streamline_training.py [-h] [-vocab_size VOCAB_SIZE] [-train_batch_size TRAIN_BATCH_SIZE] 
+        [-learning_rate LEARNING_RATE] [-num_train_epochs NUM_TRAIN_EPOCHS] [-warmup_proportion WARMUP_PROPORTION]
+        [-max_input_examples MAX_INPUT_EXAMPLES] [-train] [-export] [-use_tpu] [-gbucket GBUCKET] [-t2t T2T] 
+        [-number_layer NUMBER_LAYER] [-hidden_size HIDDEN_SIZE] [-num_attention_head NUM_ATTENTION_HEAD]
+        [-filter_size FILTER_SIZE] [-full_attention FULL_ATTENTION]
+        model_output_dir abs_path_to_lasertagger abs_path_to_bert training_file tuning_file embedding_type
+
     positional arguments:
       model_output_dir      the directory of the model output
       abs_path_to_lasertagger
-                            absolute path to the folder where the lasertagger
-                            scripts are located
-      abs_path_to_bert      absolute path to the folder where the pretrained BERT
-                            is located
+                            absolute path to the folder where the lasertagger scripts are located
+      abs_path_to_bert      absolute path to the folder where the pretrained BERT is located
       training_file         path to training samples
       tuning_file           path to tuning samples
-    
+      embedding_type        type of embedding. 
```

Comment:
add more details 

type of embedding, must be one of ["POS", ...

Reply:
Done.